packages/fluentui/docs/src/prototypes/FormValidation/FormHooks.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import { Form } from '@fluentui/react-northstar';
+import { useForm } from 'react-hook-form';
+
+const FormValidateHooks = () => {
+  const { register, handleSubmit, errors, setValue, watch, triggerValidation } = useForm({
+    mode: 'onBlur',
+  });
+
+  const onSubmit = data => {
+    console.log('errors: ', errors);
+    console.log('data: ', data);
+  };
+
+  const firstNameWatched = watch('firstName');
+
+  React.useEffect(() => {
+    register(
+      { name: 'city' },
+      {
+        required: { value: true, message: 'city is required' },
+      },
+    );
+  }, [register]);
+
+  const handleMultiChange = selectedOption => {
+    setValue('city', selectedOption);
+  };
+  return (
+    <Form onSubmit={handleSubmit(onSubmit)}>
+      <Form.Input
+        label="First Name"
+        id="first-name-hooks"
+        required
+        name="firstName"
+        value={firstNameWatched}
+        inputRef={register({
```

Comment:
```suggestion
        ref={register({
```

Why we are not using `ref` there?

Reply:
It's was already in put, changing it would be a breaking change...?

Reply:
Gonna handle it in separated PR